src/components/drawerForCreateTask/CreateNewDrawer.jsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Drawer from "@mui/material/Drawer";
+import List from "@mui/material/List";
+import Divider from "@mui/material/Divider";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import InboxIcon from "@mui/icons-material/MoveToInbox";
+import MailIcon from "@mui/icons-material/Mail";
+import style from "./drawer.module.scss";
+
+export default function CreateNewDrawer() {
+  const [state, setState] = React.useState({
```

Comment:
this is class component style, use function component style (e.g. `[right, setRigth] = React.useState(false)`)